RoddyCore/src/de/dkfz/roddy/core/Analysis.java

```diff
@@ -529,6 +508,29 @@ private void printMessages(ExecutionContext context, String datasetID, String ti
         logger.postAlwaysInfo(messages.toString());
     }
 
+    private static String getContextErrorText(ExecutionContext context) {
```

Comment:
That's also a duplicate. Actually not only a duplicate but already handled by printMessagesForContext()